styles/components-styles/nav.css

```diff
@@ -0,0 +1,77 @@
+.hamburger,
+.docs-hamburger {
+  display: none;
+  pointer-events: none;
+}
+
+.header {
+  background: var(--COLOR-PRIMARY-TRANS);
+  backdrop-filter: blur(2rem);
+  -webkit-backdrop-filter: blur(2rem);
+  box-shadow: inset 0px -1px 1px #e7ebf0;
+}
+
+.navbar {
+  width: 90%;
+  margin: auto;
+  min-height: 10vh;
+  justify-content: space-between;
+  animation: opening 1s ease-in-out forwards;
+}
+
+.nav-link-btn {
+  border-radius: unset;
+  color: var(--COLOR-PRIMARY);
+  padding: 0;
+  transition: all 0.2s ease;
+}
+
+.nav-link-btn a:hover {
+  color: var(--COLOR-COMPLEMENTARY-LIGHT);
+  border-bottom: 0.25rem solid white;
+}
+
+/* sticky header */
+.sticky-header {
+  position: sticky;
+  top: 0%;
+  z-index: 10;
+}
+
+@media screen and (max-width: 600px) {
+  .hamburger,
+  .docs-hamburger {
+    display: grid;
+    pointer-events: visible;
+    cursor: pointer;
+    color: white;
+    z-index: 3;
+  }
+
+  .docs-hamburger {
+    z-index: 0;
+  }
+
+  .nav-items {
+    --height: 40vh;
```

Comment:
i think there's a typo here, " --height ".

Reply:
Not a type :D. Here, I've used CSS variable for nav-items.. notice later I'm doing calculations for `top` alignment using this... this will position the element account to change the height.

Reply:
Okay, got it!